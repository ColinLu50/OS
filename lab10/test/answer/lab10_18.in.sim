44538
979
1027
942
979
918
