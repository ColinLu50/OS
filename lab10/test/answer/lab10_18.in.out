FCFS
71 -> 814 -> 816 -> 299 -> 255 -> 437 -> 655 -> 798 -> 64 -> 729 -> 265 -> 457 -> 630 -> 179 -> 404 -> 830 -> 175 -> 538 -> 254 -> 399 -> 645 -> 615 -> 564 -> 199 -> 914 -> 667 -> 197 -> 733 -> 360 -> 291 -> 425 -> 457 -> 646 -> 915 -> 61 -> 744 -> 131 -> 126 -> 751 -> 397 -> 105 -> 413 -> 718 -> 261 -> 410 -> 376 -> 748 -> 737 -> 482 -> 747 -> 865 -> 188 -> 693 -> 376 -> 267 -> 24 -> 449 -> 61 -> 494 -> 848 -> 294 -> 703 -> 876 -> 247 -> 62 -> 397 -> 745 -> 272 -> 406 -> 729 -> 430 -> 455 -> 749 -> 781 -> 475 -> 496 -> 935 -> 620 -> 71 -> 712 -> 615 -> 558 -> 631 -> 845 -> 597 -> 89 -> 750 -> 786 -> 506 -> 738 -> 115 -> 871 -> 101 -> 532 -> 785 -> 85 -> 781 -> 900 -> 361 -> 87 -> 904 -> 257 -> 405 -> 252 -> 525 -> 145 -> 641 -> 567 -> 663 -> 358 -> 463 -> 260 -> 263 -> 310 -> 645 -> 683 -> 497 -> 956 -> 534 -> 396 -> 443 -> 526 -> 680 -> 838 -> 249 -> 936 -> 691 -> 25 -> 875 -> 900 -> 857 -> 658 -> 350 -> 866 -> 46 -> 216 -> 117
44538
SSTF
71 -> 71 -> 64 -> 62 -> 61 -> 61 -> 46 -> 25 -> 24 -> 85 -> 87 -> 89 -> 101 -> 105 -> 115 -> 117 -> 126 -> 131 -> 145 -> 175 -> 179 -> 188 -> 197 -> 199 -> 216 -> 247 -> 249 -> 252 -> 254 -> 255 -> 257 -> 260 -> 261 -> 263 -> 265 -> 267 -> 272 -> 291 -> 294 -> 299 -> 310 -> 350 -> 358 -> 360 -> 361 -> 376 -> 376 -> 396 -> 397 -> 397 -> 399 -> 404 -> 405 -> 406 -> 410 -> 413 -> 425 -> 430 -> 437 -> 443 -> 449 -> 455 -> 457 -> 457 -> 463 -> 475 -> 482 -> 494 -> 496 -> 497 -> 506 -> 525 -> 526 -> 532 -> 534 -> 538 -> 558 -> 564 -> 567 -> 597 -> 615 -> 615 -> 620 -> 630 -> 631 -> 641 -> 645 -> 645 -> 646 -> 655 -> 658 -> 663 -> 667 -> 680 -> 683 -> 691 -> 693 -> 703 -> 712 -> 718 -> 729 -> 729 -> 733 -> 737 -> 738 -> 744 -> 745 -> 747 -> 748 -> 749 -> 750 -> 751 -> 781 -> 781 -> 785 -> 786 -> 798 -> 814 -> 816 -> 830 -> 838 -> 845 -> 848 -> 857 -> 865 -> 866 -> 871 -> 875 -> 876 -> 900 -> 900 -> 904 -> 914 -> 915 -> 935 -> 936 -> 956
979
SCAN
71 -> 64 -> 62 -> 61 -> 46 -> 25 -> 24 -> 0 -> 85 -> 87 -> 89 -> 101 -> 105 -> 115 -> 117 -> 126 -> 131 -> 145 -> 175 -> 179 -> 188 -> 197 -> 199 -> 216 -> 247 -> 249 -> 252 -> 254 -> 255 -> 257 -> 260 -> 261 -> 263 -> 265 -> 267 -> 272 -> 291 -> 294 -> 299 -> 310 -> 350 -> 358 -> 360 -> 361 -> 376 -> 396 -> 397 -> 399 -> 404 -> 405 -> 406 -> 410 -> 413 -> 425 -> 430 -> 437 -> 443 -> 449 -> 455 -> 457 -> 463 -> 475 -> 482 -> 494 -> 496 -> 497 -> 506 -> 525 -> 526 -> 532 -> 534 -> 538 -> 558 -> 564 -> 567 -> 597 -> 615 -> 620 -> 630 -> 631 -> 641 -> 645 -> 646 -> 655 -> 658 -> 663 -> 667 -> 680 -> 683 -> 691 -> 693 -> 703 -> 712 -> 718 -> 729 -> 733 -> 737 -> 738 -> 744 -> 745 -> 747 -> 748 -> 749 -> 750 -> 751 -> 781 -> 785 -> 786 -> 798 -> 814 -> 816 -> 830 -> 838 -> 845 -> 848 -> 857 -> 865 -> 866 -> 871 -> 875 -> 876 -> 900 -> 904 -> 914 -> 915 -> 935 -> 936 -> 956
1027
C-SCAN
71 -> 64 -> 62 -> 61 -> 46 -> 25 -> 24 -> 0 -> 956 -> 956 -> 936 -> 935 -> 915 -> 914 -> 904 -> 900 -> 876 -> 875 -> 871 -> 866 -> 865 -> 857 -> 848 -> 845 -> 838 -> 830 -> 816 -> 814 -> 798 -> 786 -> 785 -> 781 -> 751 -> 750 -> 749 -> 748 -> 747 -> 745 -> 744 -> 738 -> 737 -> 733 -> 729 -> 718 -> 712 -> 703 -> 693 -> 691 -> 683 -> 680 -> 667 -> 663 -> 658 -> 655 -> 646 -> 645 -> 641 -> 631 -> 630 -> 620 -> 615 -> 597 -> 567 -> 564 -> 558 -> 538 -> 534 -> 532 -> 526 -> 525 -> 506 -> 497 -> 496 -> 494 -> 482 -> 475 -> 463 -> 457 -> 455 -> 449 -> 443 -> 437 -> 430 -> 425 -> 413 -> 410 -> 406 -> 405 -> 404 -> 399 -> 397 -> 396 -> 376 -> 361 -> 360 -> 358 -> 350 -> 310 -> 299 -> 294 -> 291 -> 272 -> 267 -> 265 -> 263 -> 261 -> 260 -> 257 -> 255 -> 254 -> 252 -> 249 -> 247 -> 216 -> 199 -> 197 -> 188 -> 179 -> 175 -> 145 -> 131 -> 126 -> 117 -> 115 -> 105 -> 101 -> 89 -> 87 -> 85
942
LOOK
71 -> 64 -> 62 -> 61 -> 46 -> 25 -> 24 -> 85 -> 87 -> 89 -> 101 -> 105 -> 115 -> 117 -> 126 -> 131 -> 145 -> 175 -> 179 -> 188 -> 197 -> 199 -> 216 -> 247 -> 249 -> 252 -> 254 -> 255 -> 257 -> 260 -> 261 -> 263 -> 265 -> 267 -> 272 -> 291 -> 294 -> 299 -> 310 -> 350 -> 358 -> 360 -> 361 -> 376 -> 396 -> 397 -> 399 -> 404 -> 405 -> 406 -> 410 -> 413 -> 425 -> 430 -> 437 -> 443 -> 449 -> 455 -> 457 -> 463 -> 475 -> 482 -> 494 -> 496 -> 497 -> 506 -> 525 -> 526 -> 532 -> 534 -> 538 -> 558 -> 564 -> 567 -> 597 -> 615 -> 620 -> 630 -> 631 -> 641 -> 645 -> 646 -> 655 -> 658 -> 663 -> 667 -> 680 -> 683 -> 691 -> 693 -> 703 -> 712 -> 718 -> 729 -> 733 -> 737 -> 738 -> 744 -> 745 -> 747 -> 748 -> 749 -> 750 -> 751 -> 781 -> 785 -> 786 -> 798 -> 814 -> 816 -> 830 -> 838 -> 845 -> 848 -> 857 -> 865 -> 866 -> 871 -> 875 -> 876 -> 900 -> 904 -> 914 -> 915 -> 935 -> 936 -> 956
979
C-LOOK
71 -> 64 -> 62 -> 61 -> 46 -> 25 -> 24 -> 956 -> 936 -> 935 -> 915 -> 914 -> 904 -> 900 -> 876 -> 875 -> 871 -> 866 -> 865 -> 857 -> 848 -> 845 -> 838 -> 830 -> 816 -> 814 -> 798 -> 786 -> 785 -> 781 -> 751 -> 750 -> 749 -> 748 -> 747 -> 745 -> 744 -> 738 -> 737 -> 733 -> 729 -> 718 -> 712 -> 703 -> 693 -> 691 -> 683 -> 680 -> 667 -> 663 -> 658 -> 655 -> 646 -> 645 -> 641 -> 631 -> 630 -> 620 -> 615 -> 597 -> 567 -> 564 -> 558 -> 538 -> 534 -> 532 -> 526 -> 525 -> 506 -> 497 -> 496 -> 494 -> 482 -> 475 -> 463 -> 457 -> 455 -> 449 -> 443 -> 437 -> 430 -> 425 -> 413 -> 410 -> 406 -> 405 -> 404 -> 399 -> 397 -> 396 -> 376 -> 361 -> 360 -> 358 -> 350 -> 310 -> 299 -> 294 -> 291 -> 272 -> 267 -> 265 -> 263 -> 261 -> 260 -> 257 -> 255 -> 254 -> 252 -> 249 -> 247 -> 216 -> 199 -> 197 -> 188 -> 179 -> 175 -> 145 -> 131 -> 126 -> 117 -> 115 -> 105 -> 101 -> 89 -> 87 -> 85
918
